cmd/createtree/pem_test.go

```diff
@@ -0,0 +1,106 @@
+package main
+
+import (
+	"testing"
+
+	"github.com/golang/protobuf/ptypes"
+	"github.com/google/trillian/crypto/keys"
+	"github.com/google/trillian/crypto/keyspb"
+)
+
+func TestWithPEMKeyFile(t *testing.T) {
+	pemPath, pemPassword := "../../testdata/log-rpc-server.privkey.pem", "towel"
+
+	wantTree := *defaultTree
+	keyProto, err := ptypes.MarshalAny(&keyspb.PEMKeyFile{
+		Path:     pemPath,
+		Password: pemPassword,
+	})
+	if err != nil {
+		t.Fatalf("MarshalAny(PEMKeyFile): %v", err)
+	}
+	wantTree.PrivateKey = keyProto
+
+	runTest(t, []*testCase{
```

Comment:
Could we have runTest and testCase moved to a separate, test-library like file? I'd be ok with a testonly package inside createtree, for example.